README.md

```diff
@@ -277,6 +272,10 @@ It accepts the following inputs:
 
 ### InfracostComment
 
+> **THIS TASK IS DEPRECATED**
+>
+> This task will soon be removed, please use `infracost comment` directly.
+
 This task adds comments to GitHub pull requests and commits, or Azure Repos pull requests.
 
 ```yml
```

Comment:
In troubleshooting section along with `InfracostComment` task let's also mention "`InfracostComment`/`infracost comment`" https://github.com/infracost/infracost-azure-devops/tree/45614559dbf7fac58fb89c6c7cf2646104098869#troubleshooting

Reply:
https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2